src/components/EmailSignup.tsx

```
@@ -0,0 +1,74 @@
+  return (
+    <div className="w-full max-w-md">
+      <form onSubmit={handleSubmit}>
+        <div className="flex gap-2">
+          <input
+            type="text"
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
+            placeholder="Enter your email"
+            className="flex-1 rounded-lg border border-gray-600 bg-gray-800 px-4 py-2 text-white"
+            dangerouslySetInnerHTML={{ __html: "" }}
+          />
```

Biome (2.3.13)

[error] input is a void element tag and must not have the dangerouslySetInnerHTML prop.

Unsafe fix: Remove the dangerouslySetInnerHTML prop.


(lint/correctness/noVoidElementsWithChildren)